genesyscloud/routing_email_route/resource_genesyscloud_routing_email_route_test.go

```diff
@@ -24,142 +22,32 @@ import (
 )
 
 func TestAccResourceRoutingEmailRoute(t *testing.T) {
-
 	var (
-		domainRes          = "routing-domain1"
-		domainId           = fmt.Sprintf("terraformroutes.%s.com", strings.Replace(uuid.NewString(), "-", "", -1))
-		queueResource      = "email-queue"
-		queueName          = "Terraform Email Queue-" + uuid.NewString()
-		langResource       = "email-lang"
-		langName           = "tflang" + uuid.NewString()
-		skillResource      = "test-skill1"
-		skillName          = "Terraform Skill" + uuid.NewString()
-		routeRes           = "email-route1"
-		routeRes2          = "email-route2"
-		routePattern1      = "terraform1"
-		routePattern2      = "terraform2"
-		routePattern3      = "terraform3"
-		fromEmail1         = "terraform1@test.com"
-		fromEmail2         = "terraform2@test.com"
-		fromName1          = "John Terraform"
-		fromName2          = "Jane Terraform"
-		priority1          = "1"
-		bccEmail1          = "test1@" + domainId
-		bccEmail2          = "test2@" + domainId
-		emailFlowResource1 = "test_flow1"
-		emailFlowFilePath1 = "../../examples/resources/genesyscloud_flow/inboundcall_flow_example.yaml"
+		domainRes     = "routing-domain1"
+		domainId      = fmt.Sprintf("terraformroutes.%s.com", strings.Replace(uuid.NewString(), "-", "", -1))
+		queueResource = "email-queue"
+		queueName     = "Terraform Email Queue-" + uuid.NewString()
+		langResource  = "email-lang"
+		langName      = "tflang" + uuid.NewString()
+		skillResource = "test-skill1"
+		skillName     = "Terraform Skill" + uuid.NewString()
+		routeRes      = "email-route1"
+		routeRes2     = "email-route2"
+		routePattern1 = "terraform1"
+		routePattern2 = "terraform2"
+		routePattern3 = "terraform3"
+		fromEmail1    = "terraform1@test.com"
+		fromEmail2    = "terraform2@test.com"
+		fromName1     = "John Terraform"
+		fromName2     = "Jane Terraform"
+		priority1     = "1"
+		bccEmail1     = "test1@" + domainId
+		bccEmail2     = "test2@" + domainId
 	)
 
 	CleanupRoutingEmailDomains()
 
-	// Test error configs
-	resource.Test(t, resource.TestCase{
```

Comment:
Hi, Didnt understand why this is removed.

Reply:
reply_email_address and from_email no longer conflict with each other. The removed test cases expected a error when both are set, which now caused the test to fail.